db/push.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 type BatchPushRequest struct {
-	ClientID  string     `json:"clientId"`
+	ClientID  string     `json:"clientID"`
```

Comment:
We should go over all of these and name them consistently as `client_id`: https://w3ctag.github.io/design-principles/#casing-rules